src/components/ArticlePage.jsx

```diff
@@ -43,7 +59,37 @@ const ArticlePage = () => {
     }
   };
 
-  console.log(comments);
+  const handleCommentSubmit = (e) => {
+    e.preventDefault();
+    setIsLoadingComment(true);
+    if (commentInput === "") {
+      setNoCommentInput(true);
+      setIsLoadingComment(false);
+    } else {
+      setNoCommentInput(false);
+      postComment(article_id, commentInput, "tickle122").then((res) => {
+        setCommentInput("");
+        setIsLoadingComment(false);
+        setCommentPosted(true);
+      });
+    }
+    //hardCoded username is placeholder until login page implemented
```

Comment:
Yeah nice, useContext will come in handy there! You could even implement useContext without the login page - still have a hardcoded user just shared around your app